core/lib/types.ts

```diff
@@ -8,20 +8,43 @@ export interface Widget<
 	Actions extends object = object,
 	Directives extends object = object
 > {
+	/**
+	 * the reactive state of the widget
+	 */
 	state$: ReadableSignal<State>;
+	/**
+	 * the different stores of the widget, all reactive
```

Comment:
Maybe you can mention that the `state$` store contains all the values of the stores present in this object.

Reply:
done, thx !